### PR TITLE
[INDEX] feat: svelte indexing

### PR DESCRIPTION
**Pull request by andreancardona**
This issue closes:

https://github.com/carbon-design-system/carbon-platform/issues/516

## Description:

- This PR indexes `carbon-components-svelte`

## Note:

We still need the following thumbnails:
      

- [x] breakpoint
- [x] button-set
- [x] clickable-tile
- [x] composed-modal
- [x] context-menu
- [x] expandable-tile
- [x] fluid form
- [x] image-loader
- [x] inline-notification
- [x] local-storage
- [x] ordered-list
- [x] password-input
- [x] recursive-list
- [x] radio-tile
- [x] selectable-tile
- [x] skeleton-placeholder
- [x] skeleton-text
- [x] toast-notification
- [x] tooltip-icon
- [x] truncate
- [x] unordered-list

### CODE REVIEW

File: carbon.yml [hunk 1]
```diff
@@ -0,0 +1,752 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@v1/carbon-resources.schema.json
+library:
+  id: carbon-components-svelte
+  name: Carbon Components Svelte
+  description: Svelte implementation of Carbon Components.
+  externalDocsUrl: https://react.carbondesignsystem.com
```
Comment (francinelucca) [inline carbon.yml:6]:
```suggestion
  externalDocsUrl: https://carbon-components-svelte.onrender.com
```

File: carbon.yml [hunk 2]
```diff
@@ -0,0 +1,752 @@
...
+    externalDocsUrl: https://www.carbondesignsystem.com/components/checkbox/usage
+    framework: svelte
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/Checkbox
+  clickable-tile:
+    name: Clickable tile
+    description: ''
```
Comment (francinelucca) [inline carbon.yml:88]:
```suggestion
```

File: carbon.yml [hunk 3]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/ComboBox
+  composed-modal:
+    name: Composed modal
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:128]: should this be noIndex?

File: carbon.yml [hunk 4]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/ContentSwitcher
+  context-menu:
+    name: Context menu
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:153]: should this be noIndex?

File: carbon.yml [hunk 5]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/DatePicker
+  dropdown:
+    status: stable
+    framework: svelte
+    externalDocsUrl: https://www.carbondesignsystem.com/components/dropdown/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/DatePicker
```
Comment (francinelucca) [inline carbon.yml:197]: wrong url

File: carbon.yml [hunk 6]
```diff
@@ -0,0 +1,752 @@
...
+  expandable-tile:
+    name: Expandable tile
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:206]: should this be noIndex?

File: carbon.yml [hunk 7]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/Grid
+  image-loader:
+    name: Image loader
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:265]: should this be noIndex?

File: carbon.yml [hunk 8]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/InlineLoading
+  inline-notification:
+    name: Inline notification
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:291]: should this be noIndex?

File: carbon.yml [hunk 9]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/Loading
+  local-storage:
+    name: local-storage
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:325]: should this be noIndex?

File: carbon.yml [hunk 10]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/NumberInput
+  ordered-list:
+    name: Ordered list
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:369]: should this be noIndex?

File: carbon.yml [hunk 11]
```diff
@@ -0,0 +1,752 @@
...
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/OrderedList
+    tags:
+      - data-display
+  overflow-menu:
+    name: Overflow menu
+    description: ''
```
Comment (francinelucca) [inline carbon.yml:379]:
```suggestion
```

File: carbon.yml [hunk 12]
```diff
@@ -0,0 +1,752 @@
...
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:385]: should this be noIndex?

File: carbon.yml [hunk 13]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/PaginationNav
+  password-input:
+    name: Password input
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:418]: should this be noIndex?

File: carbon.yml [hunk 14]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/ProgressIndicator
+  radio-tile:
+    name: Radio tile
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:471]: should this be noIndex?

File: carbon.yml [hunk 15]
```diff
@@ -0,0 +1,752 @@
...
+      - input-control
+  recursive-list:
+    name: Recursive list
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:487]: should this be noIndex?

File: carbon.yml [hunk 16]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/Select
+  selectable-tile:
+    name: Selectable tile
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:522]: should this be noIndex?

File: carbon.yml [hunk 17]
```diff
@@ -0,0 +1,752 @@
...
+      - input-control
+  skeleton-placeholder:
+    name: Skeleton placeholder
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:538]: should this be noIndex?

File: carbon.yml [hunk 18]
```diff
@@ -0,0 +1,752 @@
...
+  skeleton-text:
+    name: Skeleton text
+    description: ''
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:556]: should this be noIndex?

File: carbon.yml [hunk 19]
```diff
@@ -0,0 +1,752 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/TooltipDefinition
+  tooltip-icon:
+    name: Tootlip icon
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:693]: should this be noIndex?

File: carbon.yml [hunk 20]
```diff
@@ -0,0 +1,752 @@
...
+  truncate:
+    name: Truncate
+    description: ''
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
```
Comment (francinelucca) [inline carbon.yml:718]: should this be noIndex?

File: carbon.yml [hunk 21]
```diff
@@ -0,0 +1,752 @@
...
+    status: stable
+    framework: svelte
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/TreeView
+  truncate:
+    name: Truncate
+    description: ''
```
Comment (francinelucca) [inline carbon.yml:711]:
```suggestion
```

File: carbon.yml [hunk 22]
```diff
@@ -0,0 +1,752 @@
...
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/SkeletonPlaceholder
+    #TODO: @aubrey to check tag
+    tags:
+      - shell
+  skeleton-text:
+    name: Skeleton text
+    description: ''
```
Comment (francinelucca) [inline carbon.yml:549]:
```suggestion
```

File: carbon.yml [hunk 23]
```diff
@@ -0,0 +1,752 @@
...
+    #TODO: add thumbnail
+    thumbnailPath: ''
+    noIndex: true
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/UnorderedList
+    tags:
+      - data-display
```
Comment (francinelucca) [inline carbon.yml:752]:
```suggestion
      - data-display
      
```

**Review by francinelucca — CHANGES_REQUESTED** [review]

File: carbon.yml [hunk 24]
```diff
@@ -0,0 +1,729 @@
...
+        url: https://carbon-components-svelte.onrender.com/components/ProgressIndicator
+  radio-button:
+    status: stable
+    framework: svelte
+    externalDocsUrl: https://www.carbondesignsystem.com/components/radio-button/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ProgressIndicator
```
Comment (francinelucca) [inline carbon.yml:450]: wrong url

File: carbon.yml [hunk 25]
```diff
@@ -0,0 +1,729 @@
...
+    externalDocsUrl: https://www.carbondesignsystem.com/components/tag/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/Tag
+  text-area:
+    status: stable
+    framework: svelte
+    externalDocsUrl: https://www.carbondesignsystem.com/components/text-area/usage
```
Comment (francinelucca) [inline carbon.yml:585]: this takes me to a 404 page

Comment (andreancardona) [inline carbon.yml:585]:
```suggestion
```

File: carbon.yml [hunk 26]
```diff
@@ -0,0 +1,728 @@
...
+    status: stable
+    framework: svelte
+    externalDocsUrl: https://www.carbondesignsystem.com/components/loading/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/Loading
+  local-storage:
+    name: local-storage
```
Comment (francinelucca) [inline carbon.yml:311]: is this name ok?

**Review by francinelucca — COMMENTED** [review]
LGTM 🎉 , minor comment

**Review by francinelucca — COMMENTED** [review]
LGTM

File: carbon.yml [hunk 27]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/breakpoint.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/Breakpoint
+    #TODO: @aubrey check tag
+    tags:
+      - structural-navigation
```
Comment (aubrey-oneal) [inline carbon.yml:49]: Delete structural-navigation. Add shell.

File: carbon.yml [hunk 28]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/button-set.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ButtonSet
+    #TODO: @aubrey check tag
+    tags:
+      - input-control
```
Comment (aubrey-oneal) [inline carbon.yml:74]: Approved 👍

File: carbon.yml [hunk 29]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/clickable-tile.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ClickableTile
+    #TODO: @aubrey to check
+    tags:
+      - input-control
```
Comment (aubrey-oneal) [inline carbon.yml:98]: Delete input-control. Add contextual-navigation.

File: carbon.yml [hunk 30]
```diff
@@ -0,0 +1,707 @@
...
+    thumbnailPath: './thumbnails/context-menu.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ContextMenu
+    #TODO: @aubrey to check tag
+    tags:
+      - input-control
+      - contextual-navigation
```
Comment (aubrey-oneal) [inline carbon.yml:154]: Keep input-control. Delete contextual-navigation.

File: carbon.yml [hunk 31]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/expandable-tile.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ExpandableTile
+    #TODO: @aubrey check tag
+    tags:
+      - input-control
```
Comment (aubrey-oneal) [inline carbon.yml:204]: Delete input control. Add data-display and content element (has the same tags as Accordion).

File: carbon.yml [hunk 32]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/image-loader.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ImageLoader
+    #TODO: @aubrey check tag
+    tags:
+      - shell
```
Comment (aubrey-oneal) [inline carbon.yml:259]: Keep shell. Add media tag.

File: carbon.yml [hunk 33]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/local-storage.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/LocalStorage
+    #TODO: @aubrey to check tag
+    tags:
+      - input-control
```
Comment (aubrey-oneal) [inline carbon.yml:315]: Approved 👍

File: carbon.yml [hunk 34]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/password-input.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/PasswordInput
+    #TODO: @aubrey to  check tag
+    tags:
+      - data-display
```
Comment (aubrey-oneal) [inline carbon.yml:402]: Delete data-display and contextual navigation. Add form.

File: carbon.yml [hunk 35]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/skeleton-placeholder.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/SkeletonPlaceholder
+    #TODO: @aubrey to check tag
+    tags:
+      - shell
```
Comment (aubrey-oneal) [inline carbon.yml:514]: Delete shell. Add system-feedback.

File: carbon.yml [hunk 36]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/truncate.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/Truncate
+    #TODO: @aubrey check tag
+    tags:
+      - contextual-navigation
```
Comment (aubrey-oneal) [inline carbon.yml:683]: Delete contextual-navigation. Add data-display.

**Review by aubrey-oneal — CHANGES_REQUESTED** [review]
Added changes to tags.

File: carbon.yml (hunk 27 above)
Comment (andreancardona) [inline carbon.yml:49]:
```suggestion
      - shell
```

File: carbon.yml (hunk 29 above)
Comment (andreancardona) [inline carbon.yml:98]:
```suggestion
      - contextual-navigation
```

File: carbon.yml (hunk 30 above)
Comment (andreancardona) [inline carbon.yml:154]:
```suggestion
```

File: carbon.yml (hunk 31 above)
Comment (andreancardona) [inline carbon.yml:204]:
```suggestion
      - data-display
      - content-element
```

File: carbon.yml (hunk 32 above)
Comment (andreancardona) [inline carbon.yml:259]:
```suggestion
      - shell
      - media
```

File: carbon.yml (hunk 34 above)
Comment (andreancardona) [inline carbon.yml:402]:
```suggestion
      - contextual-navigation
      - form
```

File: carbon.yml [hunk 37]
```diff
@@ -0,0 +1,707 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/recursive-list.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/RecursiveList
+    #TODO: @aubrey to check tag
+    tags:
+      - input-control
```
Comment (andreancardona) [inline carbon.yml:467]:
```suggestion
      - data-display
```

File: carbon.yml (hunk 35 above)
Comment (andreancardona) [inline carbon.yml:514]:
```suggestion
      - system-feedback
```

File: carbon.yml (hunk 36 above)
Comment (andreancardona) [inline carbon.yml:683]:
```suggestion
      - data-display
```

File: carbon.yml [hunk 38]
```diff
@@ -0,0 +1,711 @@
...
+    framework: svelte
+    thumbnailPath: './thumbnails/password-input.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/PasswordInput
+    #TODO: @aubrey to  check tag
+    tags:
+      - contextual-navigation
```
Comment (andreancardona) [inline carbon.yml:405]:
```suggestion
```

**Comment by andreancardona** [comment]
@metonym Hey Eric! Hope all is well - do you mind merging this PR when you get a chance :).

**Review by metonym — APPROVED** [review]
@andreancardona LGTM

Kudos to the designer – lovely work on the thumbnails.

**Pull request merged**